msteams-platform/task-modules-and-cards/cards/Universal-actions-for-adaptive-cards/Overview.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Overview of universal actions for adaptive cards
+description: A quick overview of universal actions for adaptive cards.
+ms.topic: overview
+---
+
+# Universal actions for adaptive cards
+
+Universal actions for adaptive cards evolved from developer feedback that even though layout and rendering for adaptive cards was universal, action handling was not. Even if a developer wanted to send the same card to different places, they would have to handle actions differently.
+
+With the Universal actions for adaptive cards, this problem is now addressed with bot becoming the common backend for handling actions along with the introduction of the new action type, `Action.Execute`, which would work across apps starting from Teams and Outlook.
+
+This document helps you to understand how you can use the universal actions for adaptive cards to enhance user experience of interacting with adaptive cards across platforms and applications.
+
+## Enhance user experience with universal actions for adaptive cards
+
+Universal actions for adaptive cards enhances user experience by enabling the following scenarios:
+
+* [Universal actions](#universal-actions)
+* [Contextual or role based views](#contextual-or-role-based-views)
+* [Sequential workflow support](#sequential-workflow-support)
+* [Up-to-date views](#up-to-date-views)
+
+### Universal actions
```

Comment:
@shiladityasaha @dipja  Can we call this by any other name? The feature itself is Universal actions for adaptive cards, so this as a scenario/sub-section seems confusing.

Reply:
This is actually outdated. It was updated